cypress/e2e/login_page.cy.js

```diff
@@ -0,0 +1,82 @@
+describe('Login Page', () => {
+    beforeEach(function(){
+
+        cy.visit('https://www.saucedemo.com/')
+    })
+
+    //Test case T-01
+    it('Check results when valid username and password is entered', () => {
+        cy.get('[data-test="username"]').type('standard_user')
+        cy.get('[data-test="password"]').type('secret_sauce')
+        cy.get('[data-test="login-button"]').click()
+
+    })
+
+    //Test case T-02
+
+    it('Check results when not valid username and password is entered', () => {
+        cy.get('[data-test="username"]').type('invalid_user')
+        cy.get('[data-test="password"]').type('secret_saucee')
+        cy.get('[data-test="login-button"]').click()
+        cy.get('[data-test="error"]')
```

Comment:
Why do you get this `'[data-test="error"]'` over here and not doing anything with it? 🤔